Lmod/branches/7.1/Lmod-illumina.spec

```diff
@@ -96,11 +96,11 @@ rm -rf $RPM_BUILD_ROOT
 %post
 SETMODULEPATH_SH="no" 
 SETMODULEPATH_CSH="no" 
-if [ -f /etc/profile.d/00-modulepath.sh ]; then
+if [ ! -f /etc/profile.d/00-modulepath.sh ]; then
```

Comment:
@spotrh: I wonder if the conditionals under the %post section of the .spec file are a potential issue or,
how to better handle that...